AMW_angular/io/src/app/resources/resource-type-add.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, EventEmitter, inject, Input, Output, Signal } from '@angular/core';
+import { ResourceType } from '../resource/resource-type';
+import { ResourceTypesService } from './resource-types.service';
+import {
+  NgbActiveModal,
+  NgbDropdown,
+  NgbDropdownItem,
+  NgbDropdownMenu,
+  NgbDropdownToggle,
+} from '@ng-bootstrap/ng-bootstrap';
+import { ModalHeaderComponent } from '../shared/modal-header/modal-header.component';
+import { ButtonComponent } from '../shared/button/button.component';
+import { FormsModule } from '@angular/forms';
+import { ResourceTypeRequest } from '../resource/resource-type-request';
+
+@Component({
+  selector: 'app-resource-type-add',
+  standalone: true,
+  imports: [
+    ModalHeaderComponent,
+    ButtonComponent,
+    FormsModule,
+    NgbDropdown,
+    NgbDropdownItem,
+    NgbDropdownMenu,
+    NgbDropdownToggle,
+  ],
+  templateUrl: './resource-type-add.component.html',
+})
+export class ResourceTypeAddComponent {
+  @Input() resourceType: ResourceType;
```

Comment:
Use input.signal

Reply:
Using InputSignal causes the error `"Resource type name must not be null or blank"` when trying to save a new Resource Type, how do I resolve this?

Reply:
Input signals are not supported with ng-bootstrap 17.0.0. 
Let's keep it with `@Input`